tkosacek/tkosacek-detroit.md

```diff
@@ -0,0 +1,5 @@
+# Detroit
+
+### Restaurants
+
```

Comment:
Mmmm Detroit, I like the Bad Brad's BBQ located at Shelby Township, I'd 120% recommend you to go to that place right after the training!